src/status_im/common/scalable_avatar/style.cljs

```diff
@@ -0,0 +1,11 @@
+(ns status-im.common.scalable-avatar.style)
+
+(defn wrapper
+  [{:keys [scale top-margin side-margin border-color]}]
```

Comment:
I think we should try avoid renaming things that already have a name.
i.e 
top-margin -> margin-top
side-margin -> margin-horizontal. Unless it needs to be descriptive in this case?